ConsoleSession.java

```diff
@@ -64,6 +66,10 @@
     private static final String CLEAR = "clear";
     private static final String EXIT = "exit";
     private static final String CLEAN = "clean";
+    private static final String KEYSPACE = "keyspaces";
```

Comment:
Also the string name and value are not identical. Perhaps you want to pluralise both?

Reply:
see above first :)